NitroxClient/Communication/MultiplayerSession/MultiplayerSessionManager.cs

```diff
@@ -43,6 +52,7 @@ public void Connect(string ipAddress, int port)
         {
             IpAddress = ipAddress;
             ServerPort = port;
+            initSerializerTask.Wait();
```

Comment:
When you merge master into this PR this method is `async` so you can use `await initSerializerTask` here.